lib/util.rb

```diff
@@ -4,6 +4,8 @@
 require "openssl"
 
 module Util
+  NFTABLES_PATH = "/home/rhizome/var/nftables"
+  NFTABLES_LOCK_FILE_PATH = "/home/rhizome/var/lock/nftables.lock"
```

Comment:
Does everything work if "/home/rhizome/var/lock/" folder is not created yet?

Reply:
that folder gets created when we bootstrap_rhizome. So, shouldn't be an issue